src/page/generate/GenerateDocumentPage.tsx

```
@@ -9,7 +9,7 @@ import {
 } from "@/components/ui/dropdown-menu"
 import { ChevronDown, MoreHorizontal } from "lucide-react"
 // import { MathpixMarkdown, MathpixLoader } from "mathpix-markdown-it"
-// import { supabase } from "@/common/supabase";
+import { supabase } from "@/common/supabase";
```

ESLint

[error] 'supabase' is defined but never used.

(@typescript-eslint/no-unused-vars)